### ISSUE TITLE
datasets used in model training

### ISSUE THREAD

**Issue by paden118** [kenshohara/3D-ResNets-PyTorch#265]
I can see from the code that for the ucf101 data set, you directly use the test set as the verification set, but the evaluation model still uses the verification set. How can you use part of the data in the training set as the verification set for training?

Look forward to your answer!

**Comment by sebyo** [comment]
@paden118 did you find a solution for your question ?
